<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Twikr</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpColor">
     <property name="title">
      <string>Window titles</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkEnabled">
        <property name="text">
         <string>&amp;Enabled</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>108</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnActive">
        <property name="text">
         <string>&amp;Active</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnInactive">
        <property name="text">
         <string>Inactive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Taskbar and Menu</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnTaskbarBg">
        <property name="text">
         <string>Taskbar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkTransparency">
     <property name="text">
      <string>Transparency</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSmallIcons">
     <property name="text">
      <string>&amp;Small icons</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>App theme</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbAppLight">
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAppDark">
        <property name="text">
         <string>Dark</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>System theme</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbSysLight">
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSysDark">
        <property name="text">
         <string>Dark</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
